libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/MapboxNavigationAccounts.kt

```diff
@@ -36,27 +36,32 @@ class MapboxNavigationAccounts private constructor() : SkuTokenProvider {
             skuGenerator = when (Billing.getInstance(context).getBillingType()) {
                 Billing.BillingModel.MAU -> MauSku(
                     preferences,
-                    TIMER_EXPIRE_AFTER * MAU_TIMER_EXPIRE_THRESHOLD
-                )
-                Billing.BillingModel.TRIPS -> TripsSku(
-                    preferences, TIMER_EXPIRE_AFTER * TRIPS_TIMER_EXPIRE_THRESHOLD,
-                    TRIPS_REQUEST_COUNT_THRESHOLD
+                    TIMER_EXPIRE_AFTER * MAU_TIMER_EXPIRE_THRESHOLD,
+                    context.applicationContext.packageName
                 )
+                Billing.BillingModel.TRIPS -> null
```

Comment:
I believe we should remove all the _Trips_ logic as we don't support that billing model anymore (even upstream in the Accounts library).

That includes https://github.com/mapbox/mapbox-navigation-android/blob/9bee8f58f0e74f1356b09ced45add5a80744cad7/libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/MapboxNavigationAccounts.kt#L17-L18 that are never used and _Trips_ related logic in [`Billing`](https://github.com/mapbox/mapbox-navigation-android/blob/master/libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/Billing.kt). What do you think?

Reply:
We would still need `BillingModel` since as per request, end client can disable MAU based billing and switch to no SKU. However, I am going to remove the constants above. Also, I think we should leave the TRIPS upstream in accounts library, as we will be soon having it in the Nav SDK.

Reply:
I didn't mean `BillingModel` I meant the specific (and never used 😅) _Trips_ logic within the `Billing` class.

> We would still need `BillingModel` since as per request, end client can disable MAU based billing and switch to no SKU.

Maybe expose a new `NO_SKU` model and remove `TRIPS` one?

Reply:
Already did that in this commit.